lib/ukallocbbuddy/bbuddy.c

```diff
@@ -355,6 +386,8 @@ static void bbuddy_pfree(struct uk_alloc *a, void *obj, unsigned long num_pages)
 
 	freed_ch->next->pprev = &freed_ch->next;
 	b->free_head[order] = freed_ch;
+
+	uk_spin_unlock(&b->level_locks[order]);
```

Comment:
The comment above (the corresponding "uk_spin_lock()" for this "uk_spin_unlock()" ) applies for this codebase logi c as well.